pkgs/top-level/all-packages.nix

```diff
@@ -34533,6 +34533,8 @@ with pkgs;
 
   osmo = callPackage ../applications/office/osmo { };
 
+  osmo-rtl-sdr = callPackage ../applications/radio/osmo-rtl-sdr { };
```

Comment:
```suggestion
```

The package should be under the `pkgs/by-name/so/some-package/package.nix` hierarchy if possible. (This is the preferred way of adding new packages when possible)

Reply:
Good call, I've moved this new package.